pkg/controller/seed/reconciler_test.go

```diff
@@ -54,6 +55,9 @@ var _ = Describe("Reconciler", func() {
 		var (
 			cosignPublicKeys []string
 			caBundle         = []byte("caBundle")
+			err              error
+			resources        map[string][]byte
```

Comment:
```suggestion
```

With `DesctibeTable` style test won't need these vars available at this scope level.